src/test/regress/expected/failure_create_index_concurrently.out

```diff
@@ -0,0 +1,153 @@
+ mitmproxy 
+-----------
+ 
+(1 row)
+
+SELECT citus.clear_network_traffic();
+ clear_network_traffic 
+-----------------------
+ 
+(1 row)
+
+CREATE INDEX CONCURRENTLY idx_index_test ON index_test(id, value_1);
+ERROR:  CONCURRENTLY-enabled index command failed
+DETAIL:  CONCURRENTLY-enabled index commands can fail partially, leaving behind an INVALID index.
+HINT:  Use DROP INDEX CONCURRENTLY IF EXISTS to remove the invalid index, then retry the original command.
+SELECT citus.dump_network_traffic();
+                                                                                                                                                                                                                                                                       dump_network_traffic                                                                                                                                                                                                                                                                        
+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+ (0,coordinator,"[initial message]")
+ (0,worker,"['AuthenticationOk()', 'ParameterStatus(application_name=citus)', 'ParameterStatus(client_encoding=UTF8)', 'ParameterStatus(DateStyle=ISO, MDY)', 'ParameterStatus(integer_datetimes=on)', 'ParameterStatus(IntervalStyle=postgres)', 'ParameterStatus(is_superuser=on)', 'ParameterStatus(server_encoding=UTF8)', 'ParameterStatus(server_version=XXX)', 'ParameterStatus(session_authorization=postgres)', 'ParameterStatus(standard_conforming_strings=on)', 'ParameterStatus(TimeZone=XXX)', 'BackendKeyData(XXX)', 'ReadyForQuery(state=idle)']")
+ (0,coordinator,"['Query(query=CREATE  INDEX CONCURRENTLY  idx_index_test_102010 ON public.index_test_XXXXXX USING btree (id ,value_1 ) )']")
+(3 rows)
+
+SELECT citus.mitmproxy('conn.allow()');
+ mitmproxy 
+-----------
+ 
+(1 row)
+
+DROP TABLE index_test;
+CREATE TABLE index_test(id int, value_1 int, value_2 int);
+SELECT create_distributed_table('index_test', 'id');
+ create_distributed_table 
+--------------------------
+ 
+(1 row)
+
+-- cancel the connection when create command is issued
+-- network traffic may differ between execution during cancellation
+-- therefore dump_network_traffic() calls are not made
+SELECT citus.mitmproxy('conn.onQuery(query="CREATE").cancel(' || pg_backend_pid() || ')');
+ mitmproxy 
+-----------
+ 
+(1 row)
+
+SELECT citus.clear_network_traffic();
```

Comment:
There's no need to call `clear_network_traffic()` if you're not going to dump afterwards. All it does it empty the buffer of captured packets.